mars_api/urls.py

```diff
@@ -5,7 +5,7 @@
 
 router = routers.DefaultRouter()
 router.register(r"players", views.PlayerViewSet)
-router.register(r"games", views.GameViewSet)
+router.register(r"games", views.GameViewSet, basename="games")
```

Comment:
Why do you need the basename?

Reply:
Have no idea why, but rest framework's front end does not provide `games` path at the index. Instead, it provided double `games_scores`

Reply:
Because you called your view `GameScoreViewSet` I think (maybe)

Reply:
![image](https://user-images.githubusercontent.com/34490649/93225527-39690780-f76a-11ea-8ec8-db9d7891590d.png)
![image](https://user-images.githubusercontent.com/34490649/93225549-3f5ee880-f76a-11ea-8b54-a9a45ecc733d.png)

Reply:
![image](https://user-images.githubusercontent.com/16212750/93432334-4d714e00-f8bd-11ea-964c-2bd002efcbff.png)

Hmmmm

Reply:
![image](https://user-images.githubusercontent.com/16212750/93432424-6bd74980-f8bd-11ea-993c-66f696ce713e.png)

HMMMMMMMMMMMMMMMMMMMM

Reply:
![image](https://user-images.githubusercontent.com/16212750/93432687-c96b9600-f8bd-11ea-9661-1e8f93076209.png)

https://media.giphy.com/media/Um3ljJl8jrnHy/giphy.gif 


I'll let you figure this one out

Reply:
According to the [Rest Framework Routers documentation](https://www.django-rest-framework.org/api-guide/routers/), it makes sense why `games` and `game_scores` have the same endpoint _displayed_: because they both use the same **queryset**.
![image](https://user-images.githubusercontent.com/34490649/93567807-24b58b00-f987-11ea-9f38-ff60da5b1e6d.png)
![image](https://user-images.githubusercontent.com/34490649/93567821-2b440280-f987-11ea-99d1-a8d7cfeab9ba.png)
![image](https://user-images.githubusercontent.com/34490649/93567912-4e6eb200-f987-11ea-8942-94b4100f8882.png)
However, I still don't get it why the endpoints' list changes when the sequence in registering does.